WcaOnRails/app/models/competition.rb

```diff
@@ -515,8 +521,12 @@ def has_entry_fee?
     base_entry_fee.nonzero?
   end
 
+  def has_competitor_limit?
+    competitor_limit_flag
```

Comment:
I don't think the UI should change. The db schema *could* change, but I don't think @larspetrus is correct about the `competitor_limit.present?` method, because `competitor_limit` is a number:

```ruby
irb(main):001:0> 0.present?
=> true
```